lib/lolcommits/plugin/loltext.rb

```diff
@@ -107,9 +115,17 @@ def annotate(image, type, string)
         transformed_position = position_transform(config_option(type, :position))
         annotate_location = '0'
 
-        # move South gravity off the edge of the image.
-        if transformed_position == 'South'
-          annotate_location = '+0+20'
+        padded_annotation = 10
+        if config_option(:border, :enabled)
+          padded_annotation = padded_annotation+config_option(:border,:size)
+        end
+
+        # move all positions off the edge of the image
+        case transformed_position
+        when "South"
+          annotate_location = "+0+#{padded_annotation}"
```

Comment:
this does change the default `annotate_location` for South, to `+0+10` instead of `+0+20` I guess if it still looks OK, thats fine..